plugins/modules/zfs.py

```diff
@@ -9,23 +9,20 @@
 from __future__ import absolute_import, division, print_function
 __metaclass__ = type
 
-DOCUMENTATION = '''
----
+DOCUMENTATION = r"""
 module: zfs
 short_description: Manage zfs
```

Comment:
While you're at it, can you fix the casing here?
```suggestion
short_description: Manage ZFS
```